modules/web/server/src/main/scala/navigate/web/server/http4s/WebServerLauncher.scala

```diff
@@ -54,19 +54,25 @@ object WebServerLauncher extends IOApp with LogInitialization {
     for
       confDir    <- baseDir[F].map(_.resolve("conf"))
       secretsConf = confDir.resolve("local").resolve("secrets.conf")
+      systemsConf = confDir.resolve("local").resolve("systems.conf")
```

Comment:
what is this? so you can override with a local file?

Reply:
Yes, this implements the same mechanism we have in observe, so that we can configure instruments in a local file.

Reply:
Would it make sense to forget about file names and just load every *.conf in the folders ?

Reply:
We could, but order matters since they override each other,